data/anaconda-gtk.css

```diff
@@ -139,3 +139,11 @@ AnacondaLayoutIndicator {
     background-color: #fdfdfd;
     color: black;
 }
+
+/* Fix the cpu-hogging spinner in progress spoke. This works by overriding the css property
+"animation-timing-function". The default Adwaita value "linear" requests infinitely fine-grained
+progress of the animation, so the component redraws every time it can. Using "steps(12)" makes the
+animation have only 12 discrete states with no interpolation, so there are effectively 12 intervals
+where the look is constant. This reduces redraws significantly and saves the CPU. See rhbz#1204242
+for measurements. */
+AnacondaStandaloneWindow spinner#progressSpinner { animation-timing-function: steps(12); }
```

Comment:
I would use 24 FPS if we are going to animate the rotation of the current icon. That shouldn't be notable at all and still improve the performance a little. If we want to use a lower number of steps, we need a different icon, that can be animated in a different way.

Please, redefine the style for all spinners of the installer.

Reply:
Done.